apps/chat/src/components/AppsEditor/GeneralInfoView/GeneralInfoView.tsx

```diff
@@ -56,11 +67,38 @@ export const GeneralInfoView: React.FC<Props> = ({
     ),
   });
 
+  const { t } = useTranslation(Translation.Chat);
+
   const formData = methods.watch();
 
+  const screenState = useScreenState();
+
+  const [previewMode, setPreviewMode] = useState<PreviewMode>(() => {
+    return screenState === ScreenState.MD
+      ? PreviewMode.closed
+      : PreviewMode.half;
+  });
```

Comment:
`screenState` can be `SM`
```suggestion
  const [previewMode, setPreviewMode] = useState<PreviewMode>(screenState <= ScreenState.MD
      ? PreviewMode.closed
      : PreviewMode.half);
```